src/libs/actions/Policy.js

```diff
@@ -248,7 +248,7 @@ function invite(logins, welcomeNote, policyID) {
     policy.alertMessage = '';
 
     // Optimistically add the user to the policy
-    Onyx.set(key, policy);
+    Onyx.merge(key, policy);
```

Comment:
I had to make this change otherwise, newly added members were being removed as soon as they were added to Onyx. 

I am not sure how to debug the Onyx but I tried debugging the code and somehow internal Keychanged handled is getting old value in the next tick.